test/unit/Formatter/JsonFormatterTest.php

```diff
@@ -76,7 +76,7 @@ public function testWrite(): void
         self::assertIsArray($data);
         self::assertEquals($expected, $data);
 
-        JsonAssert::assertJsonMatchesSchema($json, dirname(__DIR__, 3) . '/Resources/errors.schema.json');
+        JsonAssert::assertJsonMatchesSchema(json_decode($json), dirname(__DIR__, 3) . '/Resources/errors.schema.json');
```

Comment:
I think `JSON_THROWS_ON_ERROR` is needed here, no? :thinking:

Reply:
Ok, the library is a bit weird here, since it does not seem to accept primitive values at all :D 

`assert::isArray()` would suffice before this statement here, so it `array<...>` can be inferred first

Reply:
> I think JSON_THROWS_ON_ERROR is needed here, no? 

Sure, 

> assert::isArray() would suffice before this statement here, so it array<...> can be inferred first

It is a `stdClass`, Using an array will not work :/, The library cannot tell the difference between an object and array.